CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 1.3.2 - 2024-08-10
```

Comment:
best use `unreleased` in pull requests.

i am setting this directly to the correct date as i will release afterwards.

```suggestion
## 1.4.0 - 2024-09-01
```

i do this as version 1.4 - its not really a BC break but a bit of a more dangerous bugfix as some implementations might have relied on the faulty implementation